helpdesk_timesheet/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Helpdesk Timesheet",
+    "summary": """
+        Add Timesheets to the Helpdesk""",
+    "category": "Helpdesk",
+    "version": "11.0.1.0.0",
+    "depends": ["helpdesk_mgmt", "hr_timesheet"],
+    "data": [
+        "views/helpdesk_view.xml",
+        "views/hr_timesheet_view.xml",
+        "views/helpdesk_ticket_stage_view.xml",
+    ],
+    "author": "Agent ERP GmbH, " "Odoo Community Association (OCA)",
```

Comment:
One author per line?

Reply:
i use pyblack fot formatting. thats the reason why :)

Reply:
You could use:
```python
# fmt: off
...
# fmt: on
```